index.html

```diff
@@ -183,6 +183,47 @@ <h1>About</h1>
   </div>
 </section>
 
+<div class="contact-form-image">
+  <header class="header d-flex">
+    <img src="Images/geometry.png" alt="">            
+  </header>  
+</div>
+
+<section class="content d-flex" id="contact-form"> 
+  <header class="header d-flex">
+    <h1>Get started</h1>       
+  </header>  
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+    sed do eiusmod tempor incididunt ut labore et dolore magna.
+  </p> 
+  <form action="https://formspree.io/f/xwkawbvd" method="POST">
+    <fieldset class="contact-form-fieldset d-flex">
+      <label class="fieldset-label d-flex" for="user_name">
+        Your name
+        <input type="text" id="user_name" required maxlength="30" placeholder="Anastasiia" name="user_name">
+      </label>
+      <label class="fieldset-label d-flex" for="user_email">
+        Your email
+        <input type="email" id="user_email" required placeholder="I" name="user_email">
```

Comment:
- [ ] [OPTIONAL] I strongly recommend you remove the placeholder value due to you are using labels, at least you want to explain an example like:

```
placeholder="example@email.com"
```